### PR TITLE
For orgs show money spent instead of money received

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/6062

image

### CODE REVIEW

**Comment by SudharakaP** [comment]
> How about "contributed" instead of "money spent"? That would be consistent with our terminology.

Thanks. Changed to "Contributed".

File: components/search-page/SearchCollectiveCard.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ const SearchCollectiveCard = ({ collective, ...props }) => {
                     </Span>
                     {` `}
                     <Span fontSize="12px" fontWeight={400} color="black.700">
-                      <FormattedMessage defaultMessage="Money Spent" />
+                      <FormattedMessage defaultMessage="Contributed" />
```
Comment (Betree) [inline components/search-page/SearchCollectiveCard.js:105]: It's important that we reuse strings, we're reaching our quota on Crowdin

```suggestion
                      <FormattedMessage id="AmountContributed" defaultMessage="Contributed" />
```

Comment (SudharakaP) [reply components/search-page/SearchCollectiveCard.js]: Thanks. I've corrected this. 👍🏽

**Review by Betree — APPROVED** [review]
Lgtm

**Pull request merged**